x-pack/legacy/plugins/maps/public/layers/styles/vector/vector_style_defaults.js

```diff
@@ -135,7 +135,7 @@ export function getDefaultDynamicProperties() {
     [VECTOR_STYLES.FILL_COLOR]: {
       type: VectorStyle.STYLE_TYPE.DYNAMIC,
       options: {
-        color: COLOR_GRADIENTS[0].value,
+        color: undefined,
```

Comment:
Maybe `color` should not be shared state between ordinal and categorical. Maybe we should store ordinal color in `colorCategoryical` or something like that. Then you could remove the logic where you have to set the color when no value is provided in `ColorMapSelect` component. This would also have the added benefit that if a user switches from ordinal field to categorical field then there selections are not lost